src/shared/components/Layout/FiltersContainer/FiltersBar/index.tsx

```diff
@@ -0,0 +1,67 @@
+import { Typography } from "@mui/material";
+import {
+  FiltersBarContainer,
+  FilterWrapper,
+  SpecificButton,
+  StyledFiltersBar,
+  StyledInput,
+} from "./FiltersBar.styled.tsx";
+import SearchIcon from "../../../../assets/filters-icons/search.svg";
+import CalendarIcon from "../../../../assets/filters-icons/calendar.svg";
+import MinusIcon from "../../../../assets/filters-icons/minus.svg";
+import PlusIcon from "../../../../assets/filters-icons/plus.svg";
+import { useState } from "react";
+import { useIsMobileView } from "../../../../utils/useIsMobileView.tsx";
+
+export const FiltersBar = () => {
+  const isMobileView = useIsMobileView();
+  const [isSpecific, setIsSpecific] = useState(!isMobileView);
+
+  const handleSpecificButton = () => {
+    setIsSpecific(!isSpecific);
+  };
+
+  return (
+    <FiltersBarContainer>
+      {isSpecific ? (
+        <StyledFiltersBar>
```

Comment:
Create the `SpecificFilters` component to improve this code.

```
const SpecificFilters = ({ isSpecific }) => {
  if (!isSpecific) {
    return null;
  }
  // ...
}
```

and something like `GeneralFilters`
```
const GeneralFilters = ({ isSpecific }) => {
  if (isSpecific) {
    return null;
  }
  // ...
}
```
and use those in the `FiltersBar`